lib/manager/components/deployment/DeploymentConfigurationsPanel.js

```diff
@@ -31,6 +31,16 @@ const TRIP_PLANNER_VERSIONS = [
   { label: 'OTP 2.X', value: 'OTP_2' }
 ]
 
+function calculateCustomFileKey (customFile: CustomFile, idx: number): string {
+  return `${
+    customFile.filename || ''
+  }-${
+    customFile.contents || ''
+  }-${
+    customFile.uri || ''
+  }-${idx}`
```

Comment:
This formatting is kind of painful to me. OK if we "one-line" this if we just destructure the various `customFile` fields?

```suggestion
  const {contents, filename, uri} = customFile
  return `${filename || ''}-${contents || ''}-${uri || ''}-${idx}`
```